recipes/vsnp3/meta.yaml

```diff
@@ -12,35 +12,35 @@ build:
   noarch: generic
 
 source:
-  url: https://github.com/{{ user }}/{{ name }}/archive/3.01.tar.gz
+  url: https://github.com/{{ user }}/{{ name }}/archive/conda.tar.gz
```

Comment:
Can you properly tag v3.04 and point to that instead? Since `conda` is a branch, the recipe will fail as soon as you change the branch.